frontend/packages/pydantic-forms/src/components/render/Fields.tsx

```diff
@@ -35,36 +36,29 @@ const Col = ({ children }: ColProps) => {
 };
 
 export function RenderFields({ fields }: RenderFieldsProps) {
-    const { formLayout } = usePydanticFormContext();
+    const { formLayout, rhf } = usePydanticFormContext();
 
     return fields.map((field) => {
-        const FormElement = field.FormElement;
+        const element = field.FormElement;
 
-        if (!FormElement) {
+        if (!element) {
             return <></>;
         }
-
-        if (field.matchedFieldResult?.preventColRender) {
-            return (
-                <div key={field.id} e2e-id={field.id}>
-                    <FormElement field={field} />
-                </div>
-            );
-        }
+        const FormElement = wrapFieldElement(element, field, rhf);
```

Comment:
Think lower case would be preferred, we are unable to use the element in this fashion anyway: `<FormElement />`